python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -110,6 +110,10 @@ def testSubmitJobComplete(self):
         from Ganga.GPI import jobs
 
         j = jobs[-1]
+        j = j.copy()
+        j.unprepare()
+        j.prepare()
+        j.submit()
```

Comment:
I'd prefer to note this is a hack I this case. Is this related to why the payload is returned with status 127? It would be good to fix that. Is it really a persistency issue with prepared objects there should be an independent test for that.